internal/webpreview/launcher_test.go

```
@@ -0,0 +1,57 @@
+	var persisted atomic.Int32
+	s := startTestStackOpts(t, upstream, http.NotFoundHandler(), func(o *Options) {
+		o.PersistLauncherSeen = func() error {
+			persisted.Add(1)
+			return nil
+		}
+	})
+
+	if body := htmlBody(t, s); !strings.Contains(body, `"launcher_seen":false`) {
+		t.Fatalf("initial config must identify first use, got: %s", body)
+	}
+	req, _ := http.NewRequest(http.MethodPost, s.URL+LauncherSeenPath, nil)
```

golangci-lint (2.12.2)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)